EntranceShuffle.py

```diff
@@ -934,18 +933,22 @@ def validate_world(world: World, worlds: list[World], entrance_placed: Optional[
         CHILD_FORBIDDEN.append('Phantom Ganon Boss Room -> Forest Temple Before Boss')
         ADULT_FORBIDDEN.append('Phantom Ganon Boss Room -> Forest Temple Before Boss')
 
-    for entrance in world.get_shufflable_entrances():
-        if entrance.shuffled:
-            if entrance.replaces:
-                if entrance.replaces.name in CHILD_FORBIDDEN and not entrance_unreachable_as(entrance, 'child', already_checked=[entrance.replaces.reverse]):
-                    raise EntranceShuffleError('%s is replaced by an entrance with a potential child access' % entrance.replaces.name)
-                elif entrance.replaces.name in ADULT_FORBIDDEN and not entrance_unreachable_as(entrance, 'adult', already_checked=[entrance.replaces.reverse]):
-                    raise EntranceShuffleError('%s is replaced by an entrance with a potential adult access' % entrance.replaces.name)
-        else:
-            if entrance.name in CHILD_FORBIDDEN and not entrance_unreachable_as(entrance, 'child', already_checked=[entrance.reverse]):
-                raise EntranceShuffleError('%s is potentially accessible as child' % entrance.name)
-            elif entrance.name in ADULT_FORBIDDEN and not entrance_unreachable_as(entrance, 'adult', already_checked=[entrance.reverse]):
-                raise EntranceShuffleError('%s is potentially accessible as adult' % entrance.name)
+        if world.settings.logic_rules != 'glitchless': 
+            CHILD_FORBIDDEN.remove('GV Carpenter Tent -> GV Fortress Side')
+
+        for entrance in world.get_shufflable_entrances():
```

Comment:
is it intended for this block of code to be tabbed into the `if worlds[0].entrance_shuffle` condition?